defs/scenarios/storage/ceph/ceph-osd/bugs.yaml

```diff
@@ -52,3 +64,12 @@ conclusions:
         disk. Another potential impact is that it may hit hitting the race condition in Onode::put
         (see https://tracker.ceph.com/issues/56382) which will crash osds, especially in large clusters.
         You can check for the leaked bluestore_cache_other mempool in the output of "ceph daemon osd.id dump_mempools".
+  cephtracker53729:
+    decision:
+      - has_53729
+      - has_53729_osd_log
```

Comment:
As an additional suggestion, if `hotsos.core.plugins.storage.ceph.CephCrushMap.autoscaler_disabled_for_any_pool` is False then suggest to disable PG autoscaler for ALL pools so as to minimise the possibility of hitting this bug (but could still happen).

Reply:
Ok so I guess we actually need to run this check on a mon since we are basing the outcome of ceph version and config